### ISSUE TITLE
Some simple extensions to Map Stonyhurst grid plotting

### ISSUE THREAD

**Issue by wafels** [sunpy/sunpy#2308]
The example gallery shows plots where the Stonyhurst grid labels and ticklabels are inconveniently placed.  This doesn't show off the examples very well.  This could be solved, and present the casual user with some useful functionality, by having some simple Boolean switches that switch grid labels and ticklabels on off.

**Comment by talsperre** [comment]
@wafels, are you referring to this [example](http://docs.sunpy.org/en/stable/generated/gallery/2011_06_07_sampledata_overview.html)? 
In order to solve it, do we add  something like a boolean variable ```is_grid_title```, ```is_tick_label``` to ```**imshow_kwargs``` of` ```plot``` function & set the title/tick label on/off based on its value?

**Comment by wafels** [comment]
I think I was referring to earlier versions of this http://docs.sunpy.org/en/stable/generated/gallery/lat_lon_lines.html#sphx-glr-generated-gallery-lat-lon-lines-py , which is now fixed.  But the example you link to is also bad. That should be fixed.  I don't think solar longitude and latitude should be on the x and y axes at all, as it is confusing.  Some of this has been tackled by #2435.

**Comment by yudhik11** [comment]
@wafels I was going through [this page](http://docs.sunpy.org/en/stable/generated/gallery/2011_06_07_sampledata_overview.html) and found out there are issues like: the left axes in the images from the aia_cutouts are cut and are not displayed properly.

Also when I try running 
```
aia_cutout01_map = sunpy.map.Map(sample_data.AIA_193_CUTOUT01_IMAGE)
aia_cutout02_map = sunpy.map.Map(sample_data.AIA_193_CUTOUT02_IMAGE)
aia_cutout03_map = sunpy.map.Map(sample_data.AIA_193_CUTOUT03_IMAGE)
aia_cutout04_map = sunpy.map.Map(sample_data.AIA_193_CUTOUT04_IMAGE)
aia_cutout05_map = sunpy.map.Map(sample_data.AIA_193_CUTOUT05_IMAGE)

fig = plt.figure(figsize=(6, 28))
ax = fig.add_subplot(511, projection=aia_cutout01_map)
aia_cutout01_map.plot()
ax = fig.add_subplot(512, projection=aia_cutout02_map)
aia_cutout02_map.plot()
ax = fig.add_subplot(513, projection=aia_cutout03_map)
aia_cutout03_map.plot()
ax = fig.add_subplot(514, projection=aia_cutout04_map)
aia_cutout04_map.plot()
ax = fig.add_subplot(515, projection=aia_cutout05_map)
aia_cutout05_map.plot()
plt.tight_layout(pad=2.50)
plt.show()
```

the image texts overlap which is again a padding problem.
So I can have a go on this issue with @talsperre.

**Comment by StanczakDominik** [comment]
Does anything remain to do here after #2497?

**Comment by yudhik11** [comment]
Any specific? Not Sure.
If you have any more ideas which could be done I can surely give it a try.

**Comment by nabobalis** [comment]
I'm gonna close this as I think we are ok now. But if you think otherwise @wafels, we can either reopen this or a new issue.
